### DISCUSSION TITLE
[Feature Request] Can we have a helm chart?

### DISCUSSION THREAD

**Discussion by himpierre** [edumeet/edumeet#843] [Ideas]
Hi Devs.

It would be really nice to have a helm chart for deploying multiparty-meeting in k8s. You are half way there with the docker / nodejs / redis / ha proxy architecture. Could boost the usage of multiparty-meeting a lot I guess. Just a thought.

cheers, t.

**Comment by christian-2** [comment]
Have you already given thought to arranging networking for the [many ports ](https://github.com/misi/mm/blob/master/docker-compose.yml) of `mm` in the context of a realistic Kubernetes cluster (other than Minikube)? Would you impose `hostNetwork: true` everywhere?

**Comment by himpierre** [comment]
@christian-2 well I guess we have to think about it. `hostNetwork: true` is an option for sure.

http://alesnosek.com/blog/2017/02/14/accessing-kubernetes-pods-from-outside-of-the-cluster/

**Comment by misi** [comment]
Helm chart is on our roadmap. We want to separate media and signaling and web first. 

My thought to use calico and external nat for media server nodes or try to use Intel multus.
